### PR TITLE
Expand languages

### PR DESCRIPTION
**Pull request by huwd**
One Login are using TypeScript / Node / JS a lot for backend so promoting it to a first class option.

We'll need to come back around for mobile languages - but will do that in a separate PR with an eye to style guides.

image

### CODE REVIEW

File: source/standards/programming-languages.html.md.erb [hunk 1]
```diff
@@ -59,18 +58,16 @@ Our core languages for backend development are:
 - [Java](/manuals/programming-languages/java.html)
 - [Python](/manuals/programming-languages/python/python.html)
 - [Ruby](/manuals/programming-languages/ruby.html)
+- [JavaScript / Node.js](/manuals/programming-languages/nodejs/index.html)
```
Comment (huwd) [inline source/standards/programming-languages.html.md.erb:61]: Open for challenge on this... I never know what to call the thing. Do we refer to Language here -> `JavaScript` or Superset `TypeScript` or runtime `Node.js`

I use all three relatively interchangably in speech, but should probably be more rigerous in docs... the slash feels like a cop-out.

Perhaps JavaScript is  most right?

**Review by two-first-names — APPROVED** [review]

**Review by galund — COMMENTED** [review]
I think you asked a question that seemed to demand a pedantic answer, so sorry that's what you got!

Do you have a nice editor extension that's doing your curly quotes - I approve in principle if but we value consistency I wouldn't bother.

**Review by DaveLee-GDS — COMMENTED** [review]
It reads fine to me and is a worth addition. It might be out of scope for this document, but is it worth stating something about recruitment possibly being easier by way of utilising a mainstream language that is more likely covered in the candidate pool?
Thats the only comment i could think to add.

File: source/standards/programming-languages.html.md.erb (hunk 1 above)
Comment (willp-bl) [reply source/standards/programming-languages.html.md.erb:61]: TypeScript / Node.js is better, particularly as a backend language.  It's unlikely we'll want much regular JavaScript.

**Review by willp-bl — APPROVED** [review]

**Comment by huwd** [comment]
I'm going to come back around on the TypeScript point, I think I agree but I'd rather get this up and tweak after.

**Pull request merged**